### DISCUSSION TITLE
trackpad panning (instead of zoom)

### DISCUSSION THREAD

**Discussion by kujohn** [cytoscape/cytoscape.js#2917] [Q&A]
Is it possible to allow trackpad (two finger) to pan around instead of zooming? Majority of 2 finger interaction are for scrolling so it makes sense to pan/scroll around instead of zoom.

**Comment by Madd0g** [comment]
anyone got workarounds for this?

Excalidraw is a prime example where it's implemented like this, it employs pinch zooming and a two-finger gesture for panning. Which feels very natural with a trackpad
